src/PrivateDns/SetEnv.ps1

```diff
@@ -0,0 +1,22 @@
+﻿Param(
+    [parameter(Mandatory=$true)]
```

Comment:
Why do you have this? It is exactly what is in our docs.

Reply:
deleted it.